templates/library.html

```diff
@@ -52,18 +52,37 @@ <h1 style="margin-top: 00px;" id="libraryName">{{library.name}}</h1>
       <br />
       {{#selectedAssets}}
       <div data-library-version="{{version}}" class="assets library-table-container">
-          <table style="margin-top: 10px;" cellpadding="0" cellspacing="0" border="0" class="table table-striped" id="example">
-            <tbody>
-              {{#files}}
-              <tr class="library {{fileType}}-type">
-                <td class="library-column"  data-lib-name="{{library.name}}" style="position: relative;">
-                  <p class='library-url'>{{library.originalName}}/{{version}}/{{name}}</p>
-                </td>
-              </tr>
-              {{/files}}
-            </tbody>
-          </table>
+        <!-- Nav tabs -->
+        <ul class="nav nav-tabs" role="tablist">
+          {{#fileArray}}
+            <li role="presentation" {{#isActive}} class="active" {{/isActive}}><a href="#{{fileType}}" aria-controls="{{fileType}}" role="tab" data-toggle="tab">{{fileType}}</a></li>
+          {{/fileArray}}
+        </ul>
+
+        <!-- Tab panes -->
+        <div class="tab-content">
+          {{#fileArray}}
+            <div role="tabpanel" class="tab-pane {{#isActive}} active {{/isActive}}" id="{{fileType}}">
+              <table style="margin-top: 20px;" cellpadding="0" cellspacing="0" border="0" class="table table-striped" id="example">
+                <thead>
+                  <th>
+                    {{fileType}} files
+                  </th>
```

Comment:
I think we don't need to display the file type here as it's already on the tab, right?